DMTN-148.tex

```diff
@@ -50,9 +50,276 @@
+Only upon certification (the replacement for the Gen2 ingest) are the
+valid date ranges set.  To conform with the expected changes to FITS
+persistence in RFC-686, these and all other calibration-specific
+metadata fields will be upper-case.
+
+\begin{tabular}{l l l}
+  \label{tab:metadata}
+  Keyword & Inserted & Description \\
+  \hline
+  \verb|OBSTYPE| & Creation & Calibration dataset type contained in this file. \\
+  \verb|{OBSTYPE}_SCHEMA| & Creation & Storage schema for calibration.  \\
+  \verb|{OBSTYPE}_VERSION| & Creation & Storage schema version. \\
+  \verb|INSTRUME| & Creation & Instrument/camera this calibration belongs to. \\
+  \verb|DETECTOR| & Creation & Integer identifier for the ccd/detector. \\
+  \verb|FILTER| & Creation & Name of the physical filter or ``NONE''. \\
+  \verb|CALIB_ID| & Creation & DataId that can be used to request this calibration after ingest. \\
+  \verb|CALIB_CREATION_DATE| & Creation & Creation date of the calibration. \\
+  \verb|CALIB_CREATION_TIME| & Creation & Creation time of the calibration. \\
+  \verb|CALIBDATE| & Creation & Concatenation of \verb|CALIB_CREATION_DATE| and \verb|CALIB_CREATION_TIME|. \\
+  \verb|VALIDSTART| & Certification & Start date the calibration is valid from.  \\
+  \verb|VALIDEND| & Certification & End date the calibration is valid from.  \\
+\end{tabular}
+
+
+\subsection{Calibration Construction in Gen3}
+
+Calibration products should be constructed using well defined
+PipelineTask code.  This removes the need to delineate between
+``calibrations'' and ``user-curated calibrations''.  The only
+remaining difference between these two groups is what the initial
+persisted data type is: ``calibrations'' are stored in FITS images, as
+they represent corrections that are only meaningful as full images;
+``user-curated calibrations'' can be visualized in a small number of
+parameters, and so persisting these in a text format to allow user
+inspection is reasonable.  ECSV and YAML are good formats for these
+calibrations, and a calibration's readText/writeText methods should
+implement one of these.
+
+In addition to the calibration product, the PipelineTask code must
+persist a record of the input data used for construction.  This, along
+with the already persisted pipeline definition and task configuration,
+allows the calibration to be regenerated by any user, making the
+construction process much more transparent than it currently is.  A
+YAML file containing the dataId parameters for all inputs would
+satisfy this.  Appending additional construction statistics (for
+example, weight factors applied to each input during the combination
+stage) would be helpful in the validation process.
+
+Newly generated calibration products will belong to a Gen3 collection,
+and that collection will be used to manage the calibration through the
+validation and certification process.  This collection may only
+contain one entry for a given OIDF set.
```

Comment:
`OIDF` is in the acronyms, but it might be helpful to expand it at this first use.  If OIDF is unique (within a collection), then why have CALIB_ID?

Reply:
Same concept, different minds assigning a name.  I'm happy to switch all of them to `CALIB_ID`, as that's more descriptive.